src/lt/util/dom.cljs

```diff
@@ -37,92 +38,228 @@
+  Otherwise, [[append]] `child`."
+  [parent child]
   (if (.-firstChild parent)
     (.insertBefore parent child (.-firstChild parent))
     (append parent child)))
 
-(defn add-class [elem class]
+(defn add-class
+  "Add `class` to the classList of `elem`."
+  [elem class]
   (when (and elem (not (empty? (name class))))
     (.add (.-classList elem) (name class))))
 
-(defn remove-class [elem class]
+(defn remove-class
+  "Remove `class` from the classList of `elem`."
+  [elem class]
   (when (and elem (not (empty? (name class))))
     (.remove (.-classList elem) (name class))))
 
-(defn has-class? [elem class]
+(defn has-class?
+  "True when `elem` has `class` in its classList."
+  [elem class]
   (when (and elem (not (empty? (name class))))
     (.contains (.-classList elem) (name class))))
 
-(defn toggle-class [elem class]
+(defn toggle-class
+  "If `elem` has `class` then remove `class`. Otherwise, `class` is added to `elem`."
+  [elem class]
   (if (has-class? elem class)
     (remove-class elem class)
     (add-class elem class)))
 
-(defn set-css [elem things]
+(defn set-css
+  "Add each key-value pair in `things` to `elem`'s style.
+
+  Returns `nil`, if change was successful.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\" style=\"bottom: 34px;\">
+
+  ;; Returns nil, but makes the change.
+  (dom/set-css ($ \"div\") {\"bottom\" \"50px\"})
+  ;;=> nil
+  ;; div is now: <div class=\"tabsets\" style=\"bottom: 50px;\">
+  ```"
+  [elem things]
   (doseq [[k v] things]
     (aset (.-style elem) (name k) (if (keyword? v) (name v) v))))
 
-(defn css [elem things]
+(defn css
+  "If `things` is a map, sets the CSS of `elem` via [[set-css]]. Otherwise,
+  returns what is located at key `things` in `elem`.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\" style=\"bottom: 34px;\">
+
+  (css ($ \"div .tabsets\") :bottom)
+  ;;=> \"34px\"
+
+  ;; Returns nil, but makes the change.
+  (css ($ \"div .tabsets\") {\"bottom\" \"50px\"})
+  ;;=> nil
+
+  (css ($ \"div .tabsets\") \"bottom\")
+  ;;=> \"50px\"
+  ```"
+  [elem things]
   (let [things (if (= js/Object (type things))
-            (js->clj things)
-            things)]
+                 (js->clj things)
+                 things)]
     (if (map? things)
       (set-css elem things)
       (aget (.-style elem) (name things)))))
 
-(defn set-attr [elem things]
+(defn set-attr
+  "Add each key-value pair in `things` to `elem`'s attributes.
+
+  Returns `nil`, if change was successful.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\">
+
+  ;; Returns nil, but makes the change.
+  (dom/set-attr ($ \"div .tabsets\") {:draggable \"true\"})
+  ;;=> nil
+  ;; div is now: <div class=\"tabsets\" draggable=\"true\";>
+  ```"
+  [elem things]
   (doseq [[k v] things]
     (.setAttribute elem (name k) (if (keyword? v) (name v) v))))
 
-(defn attr [elem things]
+(defn attr
```

Comment:
As I didn't want to futher add to the issue pile, I created this [refactoring targets wiki](https://github.com/LightTable/LightTable/wiki/Possible-Refactoring-Targets) page for other changes.